website/source/index.html.erb

```diff
@@ -1,144 +1,434 @@
-              start Docker containers, VMs, or application runtimes like Java. Nomad
-              supports Linux, Windows, BSD and OSX, providing the flexibility to run
-              any workload.
-            </p>
+    <div class="row">
+      <div class="col-sm-offset-5 col-sm-6">
+        <h3>Automatic Machine Failures</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-5">
+        <%= inline_svg "feature-machine-failure.svg" %>
+      </div>
+      <div class="col-sm-7">
+        <p>
+          Because it knows the state of all nodes and applications, if a
+          node becomes unhealthy, Nomad will automatically reschedule the
+          applications that were previously running on the unhealthy host
+          onto a new, healthy host if capacity exists. As new clients are
+          added to the cluster, they can share the responsibility of
+          running applications and services.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section id="flexible-workloads" class="marketing">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        <span class="callout">Support</span>
+        <h2>Flexible Workloads</h2>
+        <p class="lead">
+          In addition to supporting Linux, Windows, and Mac, Nomad has
+          extensible support for containerized, virtualized, and standalone
+          applications. Easily start Docker containers, VMs, or application
+          runtimes like Java.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <%= inline_svg "feature-flexible-workloads.svg" %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-6">
+        <h3>Diverse Technologies</h3>
+        <p>
+          By supporting a wide range of technologies such as Docker, RKT, and
+          LXC, Nomad does not force you into a single technology. Easily use
+          multiple container or virtualization runtimes simultaneously.
+          Applications with their own runtime such as Java or Golang can be
+          scheduled directly without the overhead of a container.
+        </p>
+      </div>
+      <div class="col-sm-6">
+        <h3>Multi OS</h3>
+        <p>
+          The Nomad client runs on Linux, Windows, and Mac OS. This diverse
+          operating system support enables using the same application scheduler
+          for all your scheduling and runtime needs.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+
+<section id="binpacking" class="marketing black">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        <span class="callout">Maximize</span>
+        <h2>Increase Utilization &amp; Reduce Costs</h2>
+        <p class="lead">
+          Nomad uses bin packing to optimize application placement onto servers
+          to maximize resource utilization, increase density, and help reduce
```

Comment:
I'm slightly opposed to this language. Nomad itself doesn't reduce costs, and I don't think it would be fair to market it that way. A fork is a tool to help reduce messes while eating. Improper use of a fork leads to messes just as improper use of Nomad leads to unchanged or even higher costs. In other words, Nomad doesn't _cause_ a reduction in costs, therefore I don't feel comfortable using such exact language.